includes/settings/class-wcpdf-settings-debug.php

```diff
@@ -38,12 +38,13 @@ public function __construct()	{
 
 	public function output( $active_section ) {
 		$active_section = ! empty( $active_section ) ? $active_section : 'settings';
-		
+		$sections       = apply_filters( 'wpo_wcpdf_settings_debug_sections', $this->sections, $active_section );
```

Comment:
Maybe this filter should be in the constructor?

Reply:
Since the constructor method will run on the `plugins_loaded` hook, If I put the filter there, it won't work. Can a filter work in this condition?

https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/blob/a60611f709fd4952f5643e1cd23bbbde2967f56d/woocommerce-pdf-invoices-packingslips.php#L73

Reply:
@alexmigf Sorry, I think maybe you missed my previous comment, so I'm mentioning you again.

Reply:
I see. You could create a method to get them with the filter instead of setting in the constructor.

Reply:
Because otherwise, if you call the `$this->sections` in other place the filter doesn't apply.